addons/stock/tests/common.py

```diff
@@ -6,7 +6,19 @@
 from odoo.addons.product.tests.common import TestProductCommon
 
 
-class TestStockCommon(TestProductCommon):
+class TestStockCommonMixin:
+
+    @classmethod
+    def _post_setup_company(cls, companies):
```

Comment:
`TestStockCommon` class includes many other methods. When I inherit it from other test classes, it triggers errors because of clashing method names with the inheriting class.